.github/CODEOWNERS

```diff
@@ -1 +1 @@
-* @gabcoyne @madkinsz @jamiezieziula
+* @gabcoyne @madkinsz @PrefectHQ/platform
```

Comment:
GitHub says this is no good.

Reply:
it should be good now, had to add the team as a collaborator - good catch!